src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -269,18 +298,12 @@ private static bool IsTupleExpression(BoundKind kind)
         }
 
         private BoundExpression EvaluateSideEffectingArgumentToTemp(BoundExpression arg, ArrayBuilder<BoundExpression> effects,
-            ref ArrayBuilder<LocalSymbol> temps)
+            ArrayBuilder<LocalSymbol> temps)
         {
             if (CanChangeValueBetweenReads(arg, localsMayBeAssignedOrCaptured: true, structThisCanChangeValueBetweenReads: true))
             {
                 BoundAssignmentOperator store;
                 var temp = _factory.StoreToTemp(arg, out store);
-
-                if (temps == null)
```

Comment:
this is another dead code, null never reaches here,

Reply:
Nice. Thanks

---
In reply to: [217527327](https://github.com/dotnet/roslyn/pull/29645#discussion_r217527327) [](ancestors = 217527327)